modules/gapi/samples/onevpl_infer_single_roi.cpp

```diff
@@ -155,6 +156,10 @@ GAPI_OCV_KERNEL(OCVBBoxes, BBoxes) {
 
 } // namespace custom
 
+namespace cfg {
+typename cv::gapi::wip::oneVPL_cfg_param create_from_string(const std::string &line);
```

Comment:
Looking at `oneVPL_cfg_param` in the application code (where it is supposed to be used, I believe), this type name may require alignment with the rest of OpenCV (e.g. CamelCase for types)

Reply:
ok, will rename to CfgParam